synapseclient/multipart_upload.py

```diff
@@ -149,15 +152,15 @@ def _complete_multipart_upload(syn, uploadId):
 
 
 def _put_chunk(url, chunk, verbose=False):
-    response = requests.put(url, data=chunk)
-    # Make sure requests closes response stream?:
-    # see: http://docs.python-requests.org/en/latest/user/advanced/#keep-alive
     try:
```

Comment:
Not sure this is the right place to catch exceptions. Won't they be caught in the calling function _upload_chunk?